PySimpleGUI.py

```diff
@@ -3547,16 +3547,20 @@ def SetIcon(self, icon=None, pngbase64=None):
         except:
             pass
 
-    def SetMinSize(self):
-        if not all(self.MinSize):
+    def SetMinSize(self, size=None):
+        if not size:
+            size = self.MinSize
+        if not any(size):
             return None
```

Comment:
Why do the functions sometimes return values and sometimes not?

Reply:
The methods always return `None`. 
`if not any(size):  return None` means that if a user does not specify the size, whether min or max, then the method returns None and the following code will not be executed.